desub/Cargo.toml

```diff
@@ -15,7 +15,7 @@ desub-json-resolver = { path = "../desub-json-resolver/", optional = true }
 thiserror = "1.0.30"
 frame-metadata = "14.2"
 codec = { version = "2.3.1", package = "parity-scale-codec" }
-serde_json = "1.0.68"
+serde_json = { version = "1.0", features = ["preserve_order", "arbitrary_precision"] }
```

Comment:
is `preserve_order` required to work properly?

I just read it was an issue with `arbitary_precision`....